mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -212,11 +261,22 @@ def fit(self, X, y, custom_feature_names=None, groups=None, **fit_params):
         self.best_feature_names_ = None
         self.best_score_ = None
 
+        self.fixed_features = self.fixed_features
+        self.fixed_features_set_ = set()
+
         if hasattr(X, 'loc'):
             X_ = X.values
+            if self.fixed_features is not None:
+                self.fixed_features_ = tuple(X.columns.get_loc(c)
+                                             if isinstance(c, str) else c
+                                             for c in self.fixed_features
+                                             )
         else:
             X_ = X
 
+        if self.fixed_features is not None:
+            self.fixed_features_set_ = set(self.fixed_features_)
+
```

Comment:
I think you can now remove  this if-block (line 277-278), and simply do:

```
self.fixed_features_set_ = set(self.fixed_features_)
```

if `self.fixed_features_` is an empty tuple, we will get an empty set. So, all should be good here.